tests/dummy/app/styles/app.scss

```diff
@@ -34,7 +43,7 @@ html {
 
 body {
   height: 200vh;
-  background-color: $c-body-bg;
+  background-color: _e3dn('body-bg');
```

Comment:
Rather than accessing the color from e3dn, wouldn't it make more sense to define a variable and pass that variable to the map and use it here?